EIPS/2020 Block Reward Reduction.md

```diff
@@ -0,0 +1,87 @@
+---
+eip: <to be assigned>
+title: 2020 Block Reward Reduction to 0.5 ETH
+author: John Lilic @jlilic twitter.com/johnlilic johnlilicEIP@gmail.com, Jerome de Tychey @jdetychey twitter.com/jdetychey,  Others
```

Comment:
Please see [EIP-1](https://eips.ethereum.org/EIPS/eip-1) for details on author format.

> The author header optionally lists the names, email addresses or usernames of the authors/owners of the EIP. Those who prefer anonymity may use a username only, or a first name and a username. The format of the author header value must be:
```
Random J. User <address@dom.ain>

or

Random J. User (@username)

if the email address or GitHub username is included, and

Random J. User

if the email address is not given.
```

Reply:
@JLilic this field is still malformed. Please update to the following:

```suggestion
author: John Lilic (@jlilic), Jerome de Tychey (@jdetychey), Others
```